src/CovidAppointmentTable.js

```diff
@@ -1,4 +1,5 @@
 /*eslint no-unused-vars: ["error", { "varsIgnorePattern": "[iI]gnored" }]*/
+/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "[iI]gnored" }]*/
```

Comment:
is there a reason this is here twice?

Reply:
removed.  I probably pressed Ctrl-D in my IDE to see a "diff" and got a "duplicate line" instead.